For iOS I ran:

```
./bin/dotnet/dotnet build src/Controls/samples/Controls.Sample.iOS/Maui.Controls.Sample.iOS-net6.csproj -t:Run
```

And got this error:

```
  Failed to launch the simulator: Could not find file "/Users/eilonlipton/GitHub/maui/src/Controls/samples/Controls.Sample.iOS/bin/Debug/net6.0-ios/iossimulator-x64/Maui.Controls.Sample.iOS.app/Maui.Controls.Sample.iOS"
EXEC : error MT1008: Failed to launch the simulator: Could not find file "/Users/eilonlipton/GitHub/maui/src/Controls/samples/Controls.Sample.iOS/bin/Debug/net6.0-ios/iossimulator-x64/Maui.Controls.Sample.iOS.app/Maui.Controls.Sample.iOS" [/Users/eilonlipton/GitHub/maui/src/Controls/samples/Controls.Sample.iOS/Maui.Controls.Sample.iOS-net6.csproj]
          
/Users/eilonlipton/GitHub/maui/bin/dotnet/sdk/6.0.100-preview.5.21275.1/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(852,5): error MSB3073: The command "/Users/eilonlipton/GitHub/maui/bin/dotnet/packs/Microsoft.iOS.Sdk/14.5.100-preview.5.884/tools/bin/mlaunch --launchsim bin/Debug/net6.0-ios/iossimulator-x64/Maui.Controls.Sample.iOS.app/ --device ":v2:runtime=com.apple.CoreSimulator.SimRuntime.iOS-14-5,devicetype=com.apple.CoreSimulator.SimDeviceType.iPad-Pro-12-9-inch-5th-generation" --stdout /dev/ttys000 --stderr /dev/ttys000 --wait-for-exit" exited with code 1. [/Users/eilonlipton/GitHub/maui/src/Controls/samples/Controls.Sample.iOS/Maui.Controls.Sample.iOS-net6.csproj]
```

And for MacCatalyst I ran:

```
./bin/dotnet/dotnet build src/Controls/samples/Controls.Sample.MacCatalyst/Maui.Controls.Sample.MacCatalyst-net6.csproj -t:Run
```

And got this error:

```
nction=_LSOpenStuffCallLocal}
/Users/eilonlipton/GitHub/maui/bin/dotnet/sdk/6.0.100-preview.5.21275.1/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(852,5): error MSB3073: The command "open bin/Debug/net6.0-maccatalyst/maccatalyst-x64//Maui.Controls.Sample.MacCatalyst.app" exited with code 1. [/Users/eilonlipton/GitHub/maui/src/Controls/samples/Controls.Sample.MacCatalyst/Maui.Controls.Sample.MacCatalyst-net6.csproj]
```

And I tried another command line to launch MacCatalyst:

```
dotnet build src/Controls/samples/Controls.Sample.MacCatalyst/Maui.Controls.Sample.MacCatalyst-net6.csproj -t:Run -p:Configuration=Debug -p:TargetFramework=net6.0-maccatalyst --no-restore
```

And got the same error:

```
  The application cannot be opened for an unexpected reason, error=Error Domain=NSOSStatusErrorDomain Code=-10827 "kLSNoExecutableErr: The executable is missing" UserInfo={_LSLine=3665, _LSFunction=_LSOpenStuffCallLocal}
/usr/local/share/dotnet/sdk/6.0.100-preview.4.21255.9/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(881,5): error MSB3073: The command "open bin/Debug/net6.0-maccatalyst/maccatalyst-x64//Maui.Controls.Sample.MacCatalyst.app" exited with code 1. [/Users/eilonlipton/GitHub/maui/src/Controls/samples/Controls.Sample.MacCatalyst/Maui.Controls.Sample.MacCatalyst-net6.csproj]
```

Is there a different command line I should use for iOS and MacCatalyst?

--- a/eng/cake/dotnet.cake
+++ b/eng/cake/dotnet.cake
@@ -161,7 +161,7 @@ Task("VS-ANDROID")
     });
 
 Task("SAMPLE-ANDROID")
-    .Description("Provisions .NET 6 and launches an instance of Visual Studio using it.")
+    .Description("Provisions .NET 6 and launches Android Sample.")
     .IsDependentOn("dotnet")
     .IsDependentOn("dotnet-buildtasks")
     .Does(() =>
@@ -170,12 +170,21 @@ Task("SAMPLE-ANDROID")
     });
 
 Task("SAMPLE-IOS")
-    .Description("Provisions .NET 6 and launches an instance of Visual Studio using it.")
+    .Description("Provisions .NET 6 and launches launches iOS Sample.")
     .IsDependentOn("dotnet")
     .IsDependentOn("dotnet-buildtasks")
     .Does(() =>
     {
         RunMSBuildWithLocalDotNet("./src/Controls/samples/Controls.Sample.iOS/Maui.Controls.Sample.iOS-net6.csproj", deployAndRun:true);
+    });
+
+Task("SAMPLE-MAC")
+    .Description("Provisions .NET 6 and launches Mac Catalyst Sample.")
+    .IsDependentOn("dotnet")
+    .IsDependentOn("dotnet-buildtasks")
+    .Does(() =>
+    {
+        RunMSBuildWithLocalDotNet("./src/Controls/samples/Controls.Sample.MacCatalyst/Maui.Controls.Sample.MacCatalyst-net6.csproj", deployAndRun:true);
     });
 
 

--- a/eng/pipelines/handlers.yml
+++ b/eng/pipelines/handlers.yml
@@ -341,6 +341,7 @@ stages:
                   & $(DotNet.Path) build Microsoft.Maui.BuildTasks-net6.sln -c $(BuildConfiguration) -bl:$(LogDirectory)/$(BuildConfiguration)-buildtasks.binlog
                   & $(DotNet.Path) build src/Controls/samples/Controls.Sample.Droid/Maui.Controls.Sample.Droid-net6.csproj -c $(BuildConfiguration) -bl:$(LogDirectory)/$(BuildConfiguration)-android.binlog
                   & $(DotNet.Path) build src/Controls/samples/Controls.Sample.iOS/Maui.Controls.Sample.iOS-net6.csproj -c $(BuildConfiguration) -bl:$(LogDirectory)/$(BuildConfiguration)-iOS.binlog
+                  & $(DotNet.Path) build src/Controls/samples/Controls.Sample.MacCatalyst/Maui.Controls.Sample.MacCatalyst-net6.csproj -c $(BuildConfiguration) -bl:$(LogDirectory)/$(BuildConfiguration)-MacCatalyst.binlog
                   & $(DotNet.Path) build src/Controls/samples/Controls.Sample.SingleProject/Maui.Controls.Sample.SingleProject.csproj -c $(BuildConfiguration) -bl:$(LogDirectory)/$(BuildConfiguration)-SingleProject.binlog
                   & $(DotNet.Path) build Microsoft.Maui-net6.sln -c $(BuildConfiguration) -bl:$(LogDirectory)/$(BuildConfiguration).binlog
                 displayName: build samples

